src/main/scala/gameengine/GameEngine.scala

```diff
@@ -0,0 +1,109 @@
+package gameengine
+
+import characters.{GameBoard, Person, Player, Resources}
+
+import scala.Console.println
+import scala.collection.mutable.ListBuffer
+import scala.io.StdIn.readLine
+import scala.util.Random
+
+class GameEngine {
+  val resources = new Resources
+  private var questionList: List[String] = resources.listOfQuestions
+  private val random = new Random()
+  // Instantiate the game board
+  val gameBoard = new GameBoard(resources.charactersList)
+  // create firstPlayer
+  private val firstPlayer: Player = createPlayerAndAssignGameBoard()
+  private var continuedPlaying: Boolean = true
+
+  /**
+   * Create an object of player by assigning name, gameBoard and secret characters
+   * @return player object
+   */
+  def createPlayerAndAssignGameBoard(): Player = {
+    val player = new Player(name = "Player1", gameBoard = gameBoard.gameBoardForPlayer,
```

Comment:
Why did you decide to make the computer the guesser? Why not the computer has the random person and the user guesses?

Reply:
We have changed the user to ask questions and guess secret character of the CPU